src/views/auth/Signin.vue

```diff
@@ -62,6 +63,7 @@ export default class SigninView extends Vue {
         }
       }
     }
+    console.log('queryParam', paramConfig)
     return paramConfig
```

Comment:
This auto comment is an action i haven't seen before, very cool!

Reply:
Nice that it catches this. If you don't the console.log then please remove it. If you do need it, add a lint override for it (eg, `// eslint-disable-next-line no-console` on previous line).

Reply:
Fixed